__tests__/topic.test.js

```diff
@@ -0,0 +1,34 @@
+require("dotenv").config()
```

Comment:
Make sure you aren't requiring anything in that you don't need at the top of the file - I don't think you use dotenv or fs/promises in this file.